src/ui/common/Header/HeaderComponent.tsx

```diff
@@ -37,7 +37,7 @@ const HeaderComponent = (props: Props) => {
           <MenuIcon />
         </IconButton>
         <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
```

Comment:
Do not use the div to display text
for text, you can use h1 - h6, p, span

Reply:
All divs in the Topography components has been deleted as useless